### PR TITLE
databricks: add page

### PR DESCRIPTION
**Pull request by paprocki-r**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
@paprocki-r you need to sign the contributor license for this PR to go through. Please click on the link in CLAassistant's comment above and complete it.

File: pages/common/databricks.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# databricks
+
+> CLI for databricks 
+> More information: <https://docs.microsoft.com/en-us/azure/databricks/dev-tools/cli/>.
+
+- set up host (https address of your databricks environment) and token (generated from databrics' settings)
```
Comment (EmilySeville7cfg) [inline pages/common/databricks.md:6]:
```suggestion
- Set up the current host (https address of your databricks environment) and token (generated from databrics' settings)
```

File: pages/common/databricks.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+`databricks configure --token`
+
+
+- list workspace content:
```
Comment (EmilySeville7cfg) [inline pages/common/databricks.md:11]:
```suggestion
- List the current workspace content:
```

I don't see where workspace is specified so I assume we are talking about the current workspace.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Comment by CleanMachine1** [comment]
Hi, due to the fact this PR has been unattended with errors, and the CI has not been signed, I am closing this.

**Pull request closed without merging**